MAX_ACCELERATION: 1.0
MAX_YAWRATE: 0.2
MAX_D_YAWRATE: 0.35
MAX_WHEEL_ANGULAR_VELOCITY: 8.0 # 11.6
WHEEL_RADIUS: 0.125 # 11.6 and 0.125 gives max vel at ~1.45 m/s
TREAD: 0.5
